tensorflow/lite/c/common.h

```diff
@@ -225,6 +225,16 @@ void TfLiteFloatArrayFree(TfLiteFloatArray* a);
     }                                                                      \
   } while (0)
 
+#define TF_LITE_ENSURE_NEAR(context, a, b, epsilon)                    \
+  do {                                                                 \
+    auto delta = ((a) > (b)) ? ((a) - (b)) : ((b) - (a));              \
+    if (delta > epsilon) {                                             \
+      TF_LITE_KERNEL_LOG((context), "%s:%d %s not near %s (%d != %d)", \
```

Comment:
change the format specifiers to be %f? its a limitation of macros, but we only really will use _NEAR with floats and doubles, right?

Reply:
Done